app/web/src/helpers/cadPackages/jsCad/jscadWorker.ts

```diff
@@ -315,9 +315,12 @@ function parseDef(code, line) {
 }
 
 const makeScriptWorker = ({ callback, convertToSolids }) => {
-  let onInit, main, scriptStats, entities
+  let onInit, main, scriptStats, entities, lastParamsDef
```

Comment:
Should jscad handle a missing param and use the scripts default value by itself?

Reply:
Yes, that is expected behavior in jscad currently, just not properly implemented in my integration for CadHub.